src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -74,6 +100,33 @@ public override DbCommand GetCommand(DbConnection connection)
             return command;
         }
 
+        private bool VerifyCommand(WhereClause where, string input, DbConnection connection)
+        {
+            bool verifyStatus = false;
+            DbCommand command = connection.CreateCommand();
+            command.CommandText = input;
+            command.Parameters.AddRange(where.Parameters.ToArray());
+            DbDataReader reader = command.ExecuteReader();
```

Comment:
Use 
```
using(DbCommand command = connection.CreateCommand()) {
...
using(DbDataReader reader = command.ExecuteReader()) { ...
``` 
here, that will ensure that the command and reader are safely closed and disposed when done. Right now if an error occurs while reading you'll never clean up those resources.